packages/lit-html/src/lit-html.ts

```diff
@@ -341,6 +349,14 @@ export interface RenderOptions {
    * any inherited context. Defaults to the global `document`.
    */
   creationScope?: {importNode(node: Node, deep?: boolean): Node};
+  /**
+   * The initial connected state for the top-level part being rendered.
+   * The `part.setConnected()` method must be used subsequent to initial render
+   * to change the connected state of the part. Set to `false` if the initial
+   * render occurs in a disconnected tree and `AsyncDirective`s should see
+   * `isConnected === false` for their initial render.
+   */
+  isConnected?: boolean;
```

Comment:
If it's missing, the directives are considered connected? That seems odd: the default is `true` but if you include the option, you typically would pass `true`. What about `isDisconnected` or `initializeAsDisconnected`?

Reply:
The value is `??` checked, so _if_ it's provided, the value is respected, whichever way it's set. Seems less odd than flipping the polarity for the value that's used to control `isConnected`.